companion/src/firmwares/generalsettings.cpp

```diff
@@ -119,6 +119,7 @@ void GeneralSettings::init()
   backlightMode = 3; // keys and sticks
   backlightDelay = 2; // 2 * 5 = 10 secs
   inactivityTimer = 10;
+  pwrOffIfInactive = 0; // disabled
```

Comment:
Unnecessary as GeneralSettings initialised to null

Reply:
Agreed - thanks